cmd/talosctl/cmd/talos/etcd.go

```diff
@@ -99,22 +100,28 @@ var etcdMemberListCmd = &cobra.Command{
 					continue
 				}
 
-				if i == 0 {
+				for _, member := range message.Members {
+					if i == 0 {
+						if node != "" {
+							fmt.Fprintln(w, "NODE\tID\tHOSTNAME\tPEERS\tCLIENTS")
```

Comment:
I think name shouldn't be `PEERS` and `CLIENTS` as this is peer URLs of this node and client URLs of this node